playground/house2.py

```diff
@@ -0,0 +1,122 @@
+            if self.first_player_first_move == (turn == "b"):
+                order = "player"
+                stone = self.player_input(turn)
+            else:
+                order = "bot"
+                stone = self.bot.random_bot(turn)
+
+            if not self.referee.valid_check(stone, self.board.log):
+                print("invalid")
+                continue
+
+            self.board.put_stone(stone)
+            print(f"{order} stone: {stone.x},{stone.y},{turn}")
+            end_check = self.referee.end_check(self.board.log, self.board.config)
+            if end_check.is_end:
+                winner = "no one" if end_check.is_tie else order
+                print(f"{winner} wins")
+                self.board.print_winner(winner)
+                self.board.render_figure()
+                break
+
+            if render_every:
+                self.board.render_figure()
+
+    def simulate(self, p1: str, p2: str, render_every: bool = False, pause: float = 1.0, save: bool = False):
+        while True:
+            turn = util.turn_check(self.board.log, self.board.config.each_move, self.board.config.first_move)
+            if self.first_player_first_move == (turn == "b"):
+                order = p1
+                stone = self.bot_alias.get(p1, util.exit_by_alias)(self.board.log)
+            else:
+                order = p2
+                stone = self.bot_alias.get(p2, util.exit_by_alias)(self.board.log)
+
+            if not self.referee.valid_check(stone, self.board.log):
+                print("invalid")
+                continue
+
+            self.board.put_stone(stone)
+            print(f"{order} stone: {stone.x},{stone.y},{turn}")
+
+            game_info = {}
+            end_check = self.referee.end_check(self.board.log, self.board.config)
+            if end_check.is_end:
+                winner = "no one" if end_check.is_tie else order
+                print(f"{winner} wins")
+                self.board.print_winner(winner)
+
+                if save:
+                    game_info["winner"] = winner
+                    game_info["config"] = self.board.config
+                    game_info["len_log"] = len(self.board.log)
+
+                    if self.first_player_first_move:
+                        game_info["black_player"] = p1
+                        game_info["white_player"] = p2
+                    else:
+                        game_info["black_player"] = p2
+                        game_info["white_player"] = p1
+
+                    game_info["first_move"] = self.first_player_first_move
+                    game_info["log"] = self.board.log
+
+                    engine = "mysql+pymysql://root:allmeat@35.233.179.226:3306/playground_log"
+                    db = DBConfig()
+                    session = db.setup_db_connection(engine)
+                    save_game_result(session, winner=game_info['winner'], board_config=game_info['config'],
```

Comment:
we use following convention for long argument input
![Screen Shot 2020-03-09 at 8 36 58 PM](https://user-images.githubusercontent.com/16871455/76209691-d2601880-6245-11ea-9906-89d0961afcf7.png)

and use double quotation rather then single